Sming/Services/CommandProcessing/CommandHandler.cpp

```diff
@@ -179,46 +184,50 @@ void CommandHandler::processCommandOptions(String commandLine  ,CommandOutput* c
 	switch (numToken)
 	{
 		case 2 :
-			if (commandToken[1] == "help")
+			if (commandToken[1] == _F("help"))
 			{
 				printUsage = true;
 			}
-			if (commandToken[1] == "verbose")
+			if (commandToken[1] == _F("verbose"))
 			{
 				commandHandler.setVerboseMode(VERBOSE);
-				commandOutput->printf("Verbose mode selected\r\n");
+			commandOutput->print(_F("Verbose mode selected\r\n"));
 				break;
 			}
-			if (commandToken[1] == "silent")
+			if (commandToken[1] == _F("silent"))
 			{
 				commandHandler.setVerboseMode(SILENT);
-				commandOutput->printf("Silent mode selected\r\n");
+				commandOutput->print(_F("Silent mode selected\r\n"));
 				break;
 			}
 			errorCommand = true;
 			break;
 		case 3 :
-			if (commandToken[1] != "prompt")
+			if (commandToken[1] != _F("prompt"))
 			{
 				errorCommand = true;
 				break;
 			}
 			commandHandler.setCommandPrompt(commandToken[2]);
-			commandOutput->printf("Prompt set to : %s\r\n",commandToken[2].c_str());
+		commandOutput->print(_F("Prompt set to : "));
```

Comment:
Indentation of lines [212 - 214] needs adjusting to be the same as the line above.